fe/fe-core/src/main/java/com/starrocks/qe/SimpleScheduler.java

```diff
@@ -89,7 +95,51 @@ public static TNetworkAddress getHost(long backendId,
         return null;
     }
 
-    public static TNetworkAddress getHost(ImmutableMap<Long, Backend> backends,
+    public static TNetworkAddress getComputeNodeHost(ImmutableMap<Long, ComputeNode> computenodes,
+                                                     Reference<Long> computeNodeIdRef) {
```

Comment:
I think we needn't to change this class.  when query schedule, we only care the computable ability.